app/lib/handlers.dart

```diff
@@ -166,19 +168,22 @@ Future<shelf.Response> staticsHandler(shelf.Request request) async {
 
 /// Handles requests for /packages - JSON
```

Comment:
Since you pass in "basePath" now, please update the comment what URLs are coming here.